### PR TITLE
Statically mark up the ISV dashboard

### PR DESCRIPTION
**Pull request by anthonydillon**
Flesh out the mark up for the ISV dashboard

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
- Go to http://0.0.0.0:6543/isv/:flags:/blues/
- Check the page looks like the design [1]. Minus the chart which will come in the next branch.

[1] https://cloud.githubusercontent.com/assets/9105999/17696452/4f984694-63a6-11e6-8843-97c98f1fc3f9.png

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5650/
Test PASSed.

File: jujugui/static/gui/src/app/components/isv-profile/_isv-profile.scss [hunk 1]
```diff
@@ -1,3 +1,42 @@
 .isv-profile {
+  &__navigation {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/isv-profile/_isv-profile.scss]: I believe we're using 4 spaces for indentation with the css.

**Comment by hatched** [comment]
Looks good @anthonydillon Thanks

:+1: :ok:

**Comment by huwshimi** [comment]
Code :+1: thanks @anthonydillon!

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5658/
Test PASSed.

**Comment by anthonydillon** [comment]
Thanks :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**